lib/buttons-and-links/KExternalLink.vue

```diff
@@ -6,8 +6,26 @@
     :href="href"
     :download="download"
     dir="auto"
+    @mouseenter="hovering = true"
+    @mouseleave="hovering = false"
```

Comment:
too bad that this is necessary, but I think it's fine.

We could add a css transition inside `KIcon` so this uses the same fade transition as defined in buttons.scss